docs/source/api/apollo-server.mdx

```diff
@@ -237,15 +214,13 @@ Available in Apollo Server v3.4.0 and later.
 
 ##### `cache`
```

Comment:
Not really sure what category this falls into tbh.  It would be a stretch but maybe lifecycle to put it with plugins.